data/members.yml

```diff
@@ -45,6 +39,12 @@ types:
          image: /images/members/jakarta-member_primeton.png
          img_width: 155
          type: "enterprise"
+       - title: Red Hat
+         id: "redhat"
+         link: https://www.redhat.com/
+         image: /images/members/jakarta-member_redhat.svg
+         type: "strategic"
```

Comment:
You should update this type just in case it does something somewhere on the site

Reply:
Ah, yeah, good catch! Fixed!